releasing.md

```diff
@@ -1,61 +1,50 @@
-Releasing
-=========
+# Releasing
 
 ### Prerequisite: Sonatype (Maven Central) Account
 
 Create an account on the [Sonatype issues site][sonatype_issues]. Ask an existing publisher to open
 an issue requesting publishing permissions for `com.squareup` projects.
 
-
 Cutting a Release
 -----------------
```

Comment:
We don't allow user accounts anymore
```suggestion
```

Reply:
alas
<img width="1232" height="260" alt="CleanShot 2025-10-02 at 00 26 06@2x" src="https://github.com/user-attachments/assets/ff4bd25b-6b29-4d20-90b2-198f477f88f0" />